packages/migrations/src/actions/2025.01.13T10-08-00.default-role.ts

```diff
@@ -0,0 +1,11 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2025.01.13T10-08-00.default-role.ts',
+  // Adds a default role to OIDC integration
+  run: ({ sql }) => sql`
+    ALTER TABLE "oidc_integrations"
+    ADD COLUMN "default_role_id" UUID REFERENCES organization_member_roles(id)
```

Comment:
Since it has a `ON DELETE` trigger we should set an index on `"oidc_integrations"."default_role_id"`.

Reply:
Done